scripts/output.js

```diff
@@ -237,10 +237,9 @@ function downloadCSV(csvContent, old) {
 *This function...
 */
 function downloadURI(uri, name) {
-    if (uri === null || name ===null);{
-        return;
-    }
-	var safeName = name;
+    if (uri !== null) {
```

Comment:
I think we should maintain the functionality that checks if uri or name is null, and returns immediately if so. Even better, we should throw an error if we detect that either are null so the user knows what went wrong.